builder/azure/dtl/config.go

```diff
@@ -279,7 +279,7 @@ type Config struct {
 	DisallowPublicIP bool `mapstructure:"disallow_public_ip" required:"false"`
 
 	// Runtime Values
-	UserName                string
+	UserName                string `mapstructure:"user_name"`
```

Comment:
I think in this case we want to skip these runtime values from the HCL2 specs
```suggestion
	UserName                string `mapstructure-to-hcl2:",skip"`
```

Reply:
hey Sylvia, nice to meet you. If I make these two changes (for UserName and VMCreationResourceGroup) I am still getting the error when running packer validate:
```
Error: 2 errors occurred:
        * unknown configuration key: '"user_name"'
        * unknown configuration key: '"vm_creation_resource_group"'
```

Reply:
Did you regenerate the HCL2 specs? After the changes, you need to run `make generate` so that the hcl2 specs get updated accordingly.

Reply:
Indeed! Done!